src/components/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,311 @@
+					translations={{
+						over: "over",
+						remaining: "remaining",
+					}}
+				/>
+			</section>
+
+			<Drawer
+				open={defaultDrawerOpen}
+				onClose={() => toggleDrawerByName("default")}
+				title="Drawer with default behavior"
+			>
+				<p>This Drawer should only have the x close button</p>
+			</Drawer>
+
+			<Drawer
+				open={noDismissDrawerOpen}
+				onClose={() => toggleDrawerByName("no-dismiss")}
+				title="Drawer with dimiss on scrim click disabled"
+				closeOnBackgroundClick={false}
+			>
+				<p>This Drawer will not close if you click on the scrim background</p>
+			</Drawer>
+
+			<Drawer
+				open={hasBackButtonDrawerOpen}
+				onClose={() => toggleDrawerByName("back-button")}
+				onBack={() => {
+					alert("Back button pressed");
+				}}
+				title="With back button"
+			>
+				<p>This Drawer both a Back button and the x close button</p>
+			</Drawer>
+		</main>
+	);
+};
+
+export const withForm = () => {
+	const [formDrawerOpen, setFormDrawerOpen] = useState(false);
+
+	return (
+		<main>
+			<section
+				style={{
+					display: "flex",
+					justifyContent: "space-between",
+					marginBottom: "1rem",
+				}}
+			>
+				<Button onClick={() => setFormDrawerOpen(!formDrawerOpen)}>
+					Toggle Drawer Open
+				</Button>
+			</section>
+
+			<Drawer
+				open={formDrawerOpen}
+				onClose={() => setFormDrawerOpen(false)}
+				title="Submission Form"
+				closeOnBackgroundClick={false}
+			>
+				<Form
+					// biome-ignore lint/suspicious/noExplicitAny: we require maximum flexibility here
+					onSubmit={(e: any) => {
+						e.preventDefault();
+						alert("you successfully submitted");
+					}}
+					style={{ width: 300 }}
+				>
+					<p style={{ paddingBottom: 20 }}>
+						Terms of Service Example. User must accept ToS before being allowed
+						to proceed.
+					</p>
+
+					<Switch required name="ToS" value="accepted">
+						Do you accept the Terms of Service?
+					</Switch>
+
+					<section style={{ display: "flex", justifyContent: "space-between" }}>
+						<Button type="reset" variant="secondary">
+							Reset
+						</Button>
+
+						<Button type="submit">Submit</Button>
+					</section>
+				</Form>
+			</Drawer>
+		</main>
+	);
+};
+
+export const withNote = () => {
+	const [noteDrawerOpen, setNoteDrawerOpen] = useState(false);
+
+	return (
+		<main>
+			<section
+				style={{
+					display: "flex",
+					justifyContent: "space-between",
+					marginBottom: "1rem",
+				}}
+			>
+				<Button onClick={() => setNoteDrawerOpen(!noteDrawerOpen)}>
+					Toggle Notes Drawer Open
+				</Button>
+			</section>
+
+			<Drawer
+				open={noteDrawerOpen}
+				onClose={() => setNoteDrawerOpen(false)}
+				title="General Notes"
+				closeOnBackgroundClick={true}
+				width="30rem"
+			>
+				<Note>
+					<Note.Title>Meeting notes</Note.Title>
+					<Note.Content author="Kathy" self={true}>
+						Hi, can we sync?
+					</Note.Content>
+				</Note>
+				<Note>
+					<Note.Content author="Cleo" self={false}>
```

Comment:
`self` defaults to false, no need to pass it
```suggestion
					<Note.Content author="Cleo">
```

Reply:
fixed all instances